katello-attach-subscription

```diff
@@ -71,13 +72,16 @@ optparse = OptionParser.new do |opts|
   opts.on("-s", "--search=SEARCH", "search for machines matching this string only") do |s|
     @options[:search] = s
   end
-  opts.on("--use-cache", "use cache file") do
+  opts.on("--read-from-cache", "use, if possible, cache file") do
     @options[:usecache] = true
   end
-  opts.on("--cache-file=FILE", "read or write to cache file, based on --use-cache value") do |cf|
+  opts.on("--cache-file=FILE", "read or write to cache file, based on --read-from-cache value") do |cf|
     @options[:cachefile] = cf
   end
-  opts.on("-d", "--debug", "debug the script") do
+  opts.on("-v", "--verbose", "verbose output for the script") do
+    @options[:verbose] = true
+  end
+  opts.on("-d", "--debug", "debug output for the script") do
```

Comment:
I think `--debug` should just set `verbose=true` too, so you don't have to do `if verbose or debug` everywhere